app/views/menu/menu.html.erb

```diff
@@ -48,8 +48,12 @@
         <hr>
         
         <ul>
-          <% @dishes.each do |element| %>
-            <li><%= element.name %></li>
+          <% @lunch_dishes.each do |element| %>
+            <li><%= element.name%></li>
+              <% element.ingredients.each do |ingredient| %>
+                <li><%= ingredient[:portion_size] %></li>
+                <li><%= ingredient[:ingredient] %></li>
+              <% end %>
```

Comment:
I don't think ingredients should be visible on their own. Otherwise I'm gonna see water, salt, and pepper 20 times.